code/modules/admin/secrets.dm

```diff
@@ -379,6 +380,31 @@
 			for(var/obj/machinery/light/L in machines)
 				L.broken()
 
+		if("anime")
+			if(!check_rights(R_FUN))
+				return
+			feedback_inc("admin_secrets_fun_used",1)
+			feedback_add_details("admin_secrets_fun_used","CC")
+			message_admins("[key_name_admin(usr)] made everything kawaii.")
+			for(var/mob/living/carbon/human/H in mob_list)
+				H << sound('sound/AI/animes.ogg')
+
+				if(H.dna.species.id == "human")
```

Comment:
Not seeing why this is needed. There's no in code reason why you can't give other races cat stuff other than it might look a little dumb on their model. We're far past that with this though.

At the very least you could let them get the gear and honorifics